### DISCUSSION TITLE
[REQUEST] UPBGE 0.2 - Lamp Specularity Toggle API

### DISCUSSION THREAD

**Discussion by rpaladin** [UPBGE/upbge#1410] [Ideas/Features for UPBGE 0.2.5+]
As BlenderGames3D mentioned in Discord, we have yet to have a lamp function for toggling lamp specularity.
Since we already have the function **KX_LightObject.useShadow**, it would be nice to have a **KX_LightObject.useSpecular** API function too.

**Comment by youle31** [comment]
If there are interested developers, it can be a nice first patch to do taking example on what has already been done for other options.
